examples/researcher/researcher.ipynb

```diff
@@ -0,0 +1,605 @@
+   "id": "070871f0-d180-416b-a8cd-e76ff966f562",
+   "metadata": {},
+   "source": [
+    "# Metaphor Researcher\n",
+    "---\n",
+    "In this example, we will build Metaphor Researcher, a Javascript app that given a research topic, automatically searches for different sources about the topic with Metaphor and synthesizes the searched contents into a research report.\n",
+    "\n",
+    "This [interactive notebook](https://github.com/metaphorsystems/metaphor-node/tree/master/examples/researcher/researcher.ipynb) was made with the Deno Javascript kernel for Jupyter. Check out the [plain JS version](https://github.com/metaphorsystems/metaphor-node/tree/master/examples/researcher/researcher.mjs) if you prefer a regular Javascript file you can run with NodeJS, or want to skip to the final result. If you'd like to run this notebook locally, [Installing Deno](https://docs.deno.com/runtime/manual/getting_started/installation) and [connecting Deno to Jupyter](https://docs.deno.com/runtime/manual/tools/jupyter) is fast and easy.\n",
+    "\n",
+    "To play with this code, first we need a [Metaphor API key](https://dashboard.metaphor.systems/overview) and an [OpenAI API key](https://platform.openai.com/api-keys). Get 1000 Metaphor searches per month free just for [signing up](https://dashboard.metaphor.systems/overview)! \n",
+    "\n",
+    "Let's import the Metaphor and OpenAI SDKs and put in our API keys to create a client object for each.\n",
+    "\n",
+    "Make sure to pick the right imports for your runtime and paste or load your API keys."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "1e9e0882-d0ed-4c19-a404-f0957ef2b677",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "// Deno imports\n",
+    "import Metaphor from 'npm:metaphor-node';\n",
+    "import OpenAI from 'npm:openai';\n",
+    "\n",
+    "// NodeJS imports\n",
+    "//import Metaphor from 'metaphor-node';\n",
+    "//import OpenAI from 'openai';\n",
+    "\n",
+    "\n",
+    "const METAPHOR_API_KEY = // insert or load your API key here\n",
+    "const OPENAI_API_KEY = // insert or load your API key here\n",
+    "\n",
+    "const metaphor = new Metaphor(METAPHOR_API_KEY);\n",
+    "const openai = new OpenAI({ apiKey: OPENAI_API_KEY });"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af45bf85-f40d-4f27-a697-083c1fc37b08",
+   "metadata": {},
+   "source": [
+    "Since we'll be making several calls to the OpenAI API to get a completion from GPT 3.5-turbo, let's make a simple utility wrapper function so we can pass in the system and user messages directly, and get the LLM's response back as a string."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "1ad7dbb0-3732-4039-a818-fb74c79fe7f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "async function getLLMResponse({system = 'You are a helpful assistant.', user = '', temperature = 1, model = 'gpt-3.5-turbo'}){\n",
```

Comment:
for some reason when I view this in the github interface it gets cut off and i can't scroll. do you know why? 
<img width="1110" alt="Screenshot 2023-11-29 at 6 33 09 PM" src="https://github.com/metaphorsystems/metaphor-node/assets/20006225/d886ffb7-47cc-4242-b4b9-2b3429cacd70">